components/data/librarysize/requirements.txt

```diff
@@ -1,2 +1,3 @@
-maayanlab-bioinformatics @ git+https://github.com/Maayanlab/maayanlab-bioinformatics
-pandas
\ No newline at end of file
+json
```

Comment:
`json` is not a pip installable module so it doesn't go into `requirements.txt`. pandas and plotly seems correct.